mitmproxy/protocol/http.py

```diff
@@ -1,17 +1,15 @@
 from __future__ import absolute_import, print_function, division
 
-import time
-import sys
-import traceback
-
 import h2.exceptions
+import netlib.exceptions
 import six
-
+import sys
+import time
+import traceback
 from mitmproxy import exceptions
 from mitmproxy import models
 from mitmproxy.protocol import base
-
-import netlib.exceptions
+from mitmproxy.protocol import websockets as pwebsockets
```

Comment:
Just a note: I usually keep all built-in imports at the top, then 3rd party imports, and lastly our own stuff.
And I don't like the weirdly abbreviated import names like `pwebsockets` or `moptions`.